zbench.zig

```diff
@@ -152,6 +152,43 @@ pub const Results = struct {
         for (self.results) |r| try r.prettyPrint(writer, colors);
     }
 
+    /// Prints a summary output at the end of benchmarking sessions
+    /// This summary highlights the fastest benchmark by name in green, and compares each
+    /// subsequent benchmark to show how many times slower they are relative to the fastest.
+    pub fn printSummary(self: Results, writer: anytype) !void {
+        if (self.results.len == 0) return;
+
+        const bold = Color.bold.code();
+        const resetCode = Color.reset.code();
+        const greenCode = Color.green.code();
+        const blueCode = Color.blue.code();
```

Comment:
You could also either add the suffix to `bold` or drop it from the others. My tendency would be to drop it for all of them since it doesn't really add anything.